files/__addonLocation__/package.json

```diff
@@ -33,14 +33,12 @@
     "prepack": "rollup --config"
   },
   "dependencies": {
-    "@embroider/addon-shim": "^1.0.0"
+    "@embroider/addon-shim": "^1.0.0",
+    "decorator-transforms": "^1.0.1"
```

Comment:
@NullVoxPopuli ut of curiosity why would it go to `dependencies`? Is it intentional or mistake?

Reply:
because it has a tiny runtime module that needs to be imported